fix(tests): set txfee for some tbtc tests

shamardy opened the pull request:
Testnet bitcoin currently has high fees which causes some tests to fail due to not enough balance. Since `test_withdraw_segwit` tests only withdraw and not sending, we can set a fixed fee to fix this test.

mariocynicys commented:
Some tests still failing due to low tbtc balance/high fees:

macos:
- `mm2_tests::best_orders_tests::best_orders_must_return_duplicate_for_orderbook_tickers`

windows:
- `mm2_tests::best_orders_tests::test_best_orders_address_and_confirmations`
- `mm2_tests::orderbook_sync_tests::alice_can_see_the_active_order_after_orderbook_sync_segwit`
- `mm2_tests::orderbook_sync_tests::test_orderbook_segwit`

shamardy commented:
> Some tests still failing due to low tbtc balance/high fees:
> 
> macos:
> 
> * `mm2_tests::best_orders_tests::best_orders_must_return_duplicate_for_orderbook_tickers`
> 
> windows:
> 
> * `mm2_tests::best_orders_tests::test_best_orders_address_and_confirmations`
> * `mm2_tests::orderbook_sync_tests::alice_can_see_the_active_order_after_orderbook_sync_segwit`
> * `mm2_tests::orderbook_sync_tests::test_orderbook_segwit`

Fixed them in the latest commit. My initial intention was to fix these tests in a different PR but I fixed them here anyways since they are related.

mariocynicys reviewed (approved):
LGTM
